features/features.go

```diff
@@ -92,14 +84,6 @@ var (
 					enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
 
-	FeatureGateNodeSwap = newFeatureGate("NodeSwap").
-				reportProblemsToJiraComponent("node").
-				contactPerson("ehashman").
-				productScope(kubernetes).
-				enhancementPR("https://github.com/kubernetes/enhancements/issues/2400").
-				enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
-				mustRegister()
-
```

Comment:
This is enabled by default since Kubernetes 1.30: https://github.com/kubernetes/kubernetes/blob/7cd7fea30d054efe4f2cc60073a29d2c618258ec/pkg/features/kube_features.go#L1546-L1550

Should this be dropped?

Reply:
Is it locked on yet? Or does this gate existing force it down to TPNU clusters only?

Reply:
the latter, we want to keep this TPNU only